autogpt_platform/backend/test/blocks/test_example_blocks.py

```diff
@@ -0,0 +1,61 @@
+"""
+Test the example blocks to ensure they work correctly with the provider pattern.
+"""
+
+import pytest
+
+from backend.blocks.examples.example_sdk_block import ExampleSDKBlock
+from backend.blocks.examples.simple_example_block import SimpleExampleBlock
+from backend.sdk import APIKeyCredentials, SecretStr
+
+
+class TestExampleBlocks:
+    """Test the example blocks."""
+
+    @pytest.mark.asyncio
+    async def test_simple_example_block(self):
+        """Test the simple example block."""
+        block = SimpleExampleBlock()
+
+        # Test execution
+        outputs = {}
+        async for name, value in block.run(
+            SimpleExampleBlock.Input(text="Hello ", count=3),
+        ):
+            outputs[name] = value
+
+        assert outputs["result"] == "Hello Hello Hello "
+
+    @pytest.mark.asyncio
+    async def test_example_sdk_block(self):
+        """Test the example SDK block with credentials."""
+        # Create test credentials
+        test_creds = APIKeyCredentials(
+            id="test-creds",
```

Comment:
this should fail lmao the id needs to be a uuid (ideally unique)

Reply:
removed